http/cors.go

```diff
@@ -0,0 +1,8 @@
+package http
+
+import "github.com/rs/cors"
+
+func (ar *apiRouter) initCORS(o cors.Options) {
```

Comment:
I'm not sure it's worth to extract it to the external function. It's just two lines of code.